lib/pluginsync/util.rb

```diff
@@ -139,8 +140,8 @@ def self.html_list(list)
       case list
       when ::Array
         if list.size > 1
-          items = list.collect{|i| "<li>#{i}</li>"}.join('')
-          "<ul>#{items}</ul>"
+          items = list.collect{|i| "#{i}"}.join(' ')
+          "#{items}"
```

Comment:
You can omit this line if you don't need to wrap everything around <ul> <li>. In fact we don't even need the temp variable items or collect. These two lines can simply be:
```ruby
list.join(' ')
```